CHANGELOG.asciidoc

```diff
@@ -7,6 +7,12 @@
 === Beats version 8.15.1
 https://github.com/elastic/beats/compare/v8.15.0\...v8.15.1[View commits]
 
+==== Breaking changes
+
+*Metricbeat*
+
+- Add GCP organization and project details to ECS cloud fields. {pull}40461[40461]
```

Comment:
This seems to be in the 8.15.1 section of the release notes, but I guess we're shipping in 8.15.2 at the earliest, right?